### ISSUE TITLE
Feature Request: Introduce an option to allow old pane navigation algo

### ISSUE THREAD

**Issue by heiselph** [tmux/tmux#1607]
I'm on Linux, tmux2.6 but I don't think it matters.

TL;DR: Requesting to get an option to allow the old pane navigation that uses top or left in ambiguous cases rather than the new MRU algo.

Introduced in this commit AFAICT: https://github.com/tmux/tmux/commit/540f0b3e45acae9b0c5fa12f7e9170c6c78d177f#diff-edea1d05eae4f5d2ad664f909d26e618

I use a pane layout that involves ambiguous pane movements: two wide and short panes on top split in the middle for running commands, three vertically split long panes below for code.

Ever since a version update years ago, I've been inefficient at moving panes because I don't remember which pane I'll go to in an ambiguous situation. With the old way, it was "deterministic" and I could always quickly navigate.

Would you be willing to consider an option for this? I would be happy to contribute the code if that is agreeable.

Thank you for considering!

**Comment by nicm** [comment]
Hi

We are not going to add an option for this, sorry. tmux has a lot of
options already and we are are not adding any more obscure options that
only one or two people will ever use.

It should be possible to write a script to navigate however you want and
use that instead of select-pane -LRUD. Or you could patch your own copy of
tmux of course.

**Comment by heiselph** [comment]
Understood. Thanks for the reply!
